docs/blog/2024-09-12-next-gen-module-federation-deployments.md

```diff
@@ -0,0 +1,549 @@
+---
+title: Next-Gen Module Federation Deployments with Nx and Zephyr Cloud
+slug: next-gen-module-federation-deployment
+authors: ['Colum Ferry']
+tags: [nx, module federation, rspack, zephyr cloud]
+cover_image: /blog/images/2024-09-10/personal-access-tokens-header.avif
+published: false
+---
+
+Nx has supported Module Federation for a long time now and its popularity is seeing year-on-year increase. It has proven
+to be the defacto solution for modern Micro Frontends and Nx has embraced this and helped developers scaffold projects
+that use Module Federation; offering the most scalable and the most developer friendly experience to do so.
+However, we have always taken a step back when it came to the final piece of the puzzle. Deployment.
```

Comment:
```suggestion
However, we have always taken a step back when it came to the final piece of the puzzle: **Deployment**.
```

Reply:
Maybe also make a paragraph cut here and continue with why deployment is complex etc.